share/ext/glfw/build_glfw.sh

```diff
@@ -38,6 +38,9 @@ CMAKE_FLAGS="-DCMAKE_BUILD_TYPE=Release ${CMAKE_FLAGS}"
 CMAKE_FLAGS="-DGLFW_BUILD_EXAMPLES=OFF ${CMAKE_FLAGS}"
 CMAKE_FLAGS="-DGLFW_BUILD_TESTS=OFF ${CMAKE_FLAGS}"
 CMAKE_FLAGS="-DGLFW_BUILD_DOCS=OFF ${CMAKE_FLAGS}"
+CMAKE_FLAGS="-DGLFW_VULKAN_STATIC=OFF ${CMAKE_FLAGS}"
+
+CMAKE_FLAGS_VULKAN="-DGLFW_VULKAN_STATIC=ON"
```

Comment:
We build two libraries, one with vulkan support and one without

Reply:
This is also why the packages are a lot bigger. Not sure if we can do anything about it since we want vulkan as opt-in/opt-out